packages/mattermost-redux/src/action_types/admin.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import keyMirror from 'mattermost-redux/utils/key_mirror';
```

Comment:
Note that the import paths within mattermost-redux all had to be rewritten to start with the package name. The alias let us avoid that with all the webapp code, but it's needed in here. This can hopefully be reverted once the monorepo is properly set up, but it's necessary for now.